derive_builder/src/options/struct_mode.rs

```diff
@@ -70,14 +70,23 @@ impl OptionsBuilderMode for StructMode {
 
 impl From<OptionsBuilder<StructMode>> for (StructOptions, OptionsBuilder<FieldMode>) {
     fn from(b: OptionsBuilder<StructMode>) -> (StructOptions, OptionsBuilder<FieldMode>) {
+        let field_default = match b.default_expression {
+            Some(DefaultExpression::Trait) 
```

Comment:
Never mind; the initializer still needs some way of differentiating between "use the `Default` trait" and "use the `__default` object"